rest-api-spec/src/main/resources/rest-api-spec/test/nodes.reload_secure_settings/10_basic.yml

```diff
@@ -1,8 +1,29 @@
 ---
-"node_reload_secure_settings test":
+"node_reload_secure_settings test wrong password":
 
   - do:
-      nodes.reload_secure_settings: {}
+      nodes.reload_secure_settings:
+        body:
+          secure_settings_password: awrongpasswordhere
+  - set:
+    nodes._arbitrary_key_: node_id
 
   - is_true: nodes
   - is_true: cluster_name
+  - match: { nodes.$node_id.reload_exception.type: "security_exception" }
+  - match: { nodes.$node_id.reload_exception.reason: "Provided keystore password was incorrect" }
+
```

Comment:
Can we have a version with no password as well?
That should fail with the same error, but I think it's worth a test that it's not failing on "password is required" or a NPE, etc.